spec/firebase_app_distribution_action_spec.rb

```diff
@@ -26,6 +26,23 @@
     stubs.verify_stubbed_calls
     Faraday.default_connection = nil
   end
+  describe '#get_value_from_value_or_file' do
```

Comment:
Should this be move to a different spec for just the helper module?

Reply:
We were thinking of moving it if the helper method gets bigger and requires more tests.